scripts/bountiful.ash

```diff
@@ -463,11 +464,16 @@ boolean hunt_bounty(bounty b) {
       use_familiar($familiar[Nosy Nose]);
 
     // unlock special zone if currently not available
-    if(!can_adv(b.location, false))
-      buy(1, CONTENT_ITEMS[b.location], maxSpecial);
+    if(!can_adventure(b.location)) {
+      print("Mafia determined we can't adv at " + b.location, "blue");
+      print("Attempting to unlock.", "blue");
+      if(item_amount(CONTENT_ITEMS[b.location]) < 1)
```

Comment:
overall, logic looks sound, and as it worked in testing i lean towards merging. but i do have one very small note here. do we want to add some sort of "in-run" catch here? for instance, if they're in a HC run, they won't be able to get a content unlocker period. i could see some folks wanting to use this script during runs to just finish bounties and work towards olfaction, but i could see the canadv failure potentially causing a script blip. feel free to simply add this as a new issue rather than solving it in this PR, this looks fine, but i would appreciate it if someone could test this in-run to see what the script's behavioral path is if it cannot unlock the zone properly.

Reply:
Good call out. Made issue #4 for it it. I'll put an alt in a HC ed run and we'll see what happens